seacatauth/authn/service.py

```diff
@@ -81,8 +81,8 @@ def __init__(self, app, service_name='seacatauth.AuthenticationService'):
 		self.LoginSessions = {}
 
 		# Metrics - login counters
-		self.LoginFailedCounter = self.MetricsService.create_counter("login_failed", tags={"help": "Counts failed logins.", "unit": "logins", "default_label":"SeaCatAuth Metrics"}, init_values={"logins": 0})
-		self.LoginSuccessCounter = self.MetricsService.create_counter("login_success", tags={"help": "Counts successful logins.", "unit": "logins", "default_label":"SeaCatAuth Metrics"}, init_values={"logins": 0})
+		self.LoginFailedCounter = self.MetricsService.create_counter("login_failed", tags={"help": "Counts failed logins.", "unit": "logins", "default_label": "SeaCatAuth_Metrics"}, init_values={"logins": 0})
+		self.LoginSuccessCounter = self.MetricsService.create_counter("login_success", tags={"help": "Counts successful logins.", "unit": "logins", "default_label": "SeaCatAuth_Metrics"}, init_values={"logins": 0})
```

Comment:
Is the label specific enough?

Reply:
Not at all. It's one of my many questions. I'd appreciate a broader discussion.

Reply:
Ok - can you provide a sample of the output then.

Reply:
It sucks. But naming is just part of where I am stuck right now and the last outcome of several more questions... 
```
# TYPE active_sessions_sessions_sessions gauge
# UNIT active_sessions_sessions_sessions sessions
# HELP active_sessions_sessions_sessions Counts active sessions.
active_sessions_sessions_sessions{default_label="SeaCatAuth_Metrics"} 2
# TYPE provider1_credentials_users gauge
# UNIT provider1_credentials_users users
# HELP provider1_credentials_users Counts credentials.
provider1_credentials_users{default_label="SeaCatAuth_Metrics"} 4
# TYPE provider2_credentials_users gauge
# UNIT provider2_credentials_users users
# HELP provider2_credentials_users Counts credentials.
provider2_credentials_users{default_label="SeaCatAuth_Metrics"} 0
# TYPE login_failed_logins_logins counter
# UNIT login_failed_logins_logins logins
# HELP login_failed_logins_logins Counts failed logins.
login_failed_logins_logins_total{default_label="SeaCatAuth_Metrics"} 8
# TYPE login_success_logins_logins counter
# UNIT login_success_logins_logins logins
# HELP login_success_logins_logins Counts successful logins.
login_success_logins_logins_total{default_label="SeaCatAuth_Metrics"} 0
# EOF
```